apollo/manifest.json

```diff
@@ -33,7 +33,7 @@
         "creates_events": false
       },
       "metrics": {
-        "prefix": "apollo.",
+        "prefix": "apollo.operations.",
```

Comment:
I'm curious if the manifest supports having an array here?  It would seem safer conceptually to me — to preserve behavior of our legacy `apollo.engine.` namespace — to have this be `["apollo.operations.", "apollo.engine."`]` and _not_ add `apollo.router.` to that list (which is the primary set we're trying to unblock tag-editing access for).